projects/angular-calendar/src/modules/common/keydown-enter.directive.ts

```diff
@@ -1,17 +1,46 @@
-import { Directive, Output, EventEmitter, HostListener } from '@angular/core';
+import {
+  Directive,
+  Output,
+  EventEmitter,
+  ElementRef,
+  NgZone,
+  Renderer2,
+  OnDestroy,
+} from '@angular/core';
 
 @Directive({
   selector: '[mwlKeydownEnter]',
 })
-export class KeydownEnterDirective {
-  @Output('mwlKeydownEnter') keydown = new EventEmitter<any>(); // tslint:disable-line
+export class KeydownEnterDirective implements OnDestroy {
+  @Output('mwlKeydownEnter') keydown = new EventEmitter<KeyboardEvent>(); // tslint:disable-line
```

Comment:
The any type here is deliberate, as adding this breaks compatibility with angular 6 and 7 which are still supported (https://github.com/mattlewis92/angular-calendar/commit/d64138fe1684b9ede34af7d15d978a98166cc6ce)

Reply:
Done